src/scss/use-cases/_general.scss

```diff
@@ -33,8 +56,11 @@
 
 	&:focus {
 		color: oColorsGetPaletteColor($hoverColor);
-		outline: 2px solid oColorsGetPaletteColor($outlineColor);
 		border-bottom-color: transparent;
+
+		@if $outlineColor {
```

Comment:
would this fall under oNormalise in the future?

Reply:
Yeah, I've created an issue to remove this: https://github.com/Financial-Times/o-typography/pull/168/files/430810a0647817edb5b045bdfd7f8dcadd1c7db0#diff-98f8d72bdcc9ce07dd693c76458fc281R45